server/src/app.js

```diff
@@ -3,8 +3,7 @@ import express from 'express';
 import bodyParser from 'body-parser';
 import cors from 'cors';
 import chalk from 'chalk';
-import { db } from './models/index';
-import dbConn from './startup/dbConnection';
+import routes from './routes/index';
```

Comment:
Index is not necessary, by default it will use index './routes' is better, apply the same for other files